packages/mobile/src/features/videos/components/SingleScreen/DescriptionLeft.tsx

```diff
@@ -35,12 +35,16 @@ export class DescriptionLeft extends React.Component<any,DesLeft> {
   )
 
   componentDidMount(){
-    const listVideos =  GetVideosListExpandedV();
+    const listVideos:any =  GetVideosListExpandedV();
     this.setState ({
       listVideos
     })
   }
 
+  desLeft = (item:any)=>{
+    return null;
```

Comment:
co to?

Reply:
a to nie jest mój widok, tylko w nim był jakiś bug i go poprawialiśmy razem :)